### ISSUE TITLE
Feature request: icon from URL

### ISSUE THREAD

**Issue by retorquere** [kofigumbs/multi#129]
This is mainly a papercuts issue (I've just found multi and it's a godsent) but it would be neat if I can just give an URL for the icon rather than having to download and select it.

**Comment by kofigumbs** [comment]
Makes sense, and thanks for trying Multi! If #66 were implemented, would that fit your use case as well?

**Comment by retorquere** [comment]
If that fetches the favicon via URL, then yes, that would be perfect. I'd usually be looking for the favicon URL in any case.
